features/step_definitions/datetime_steps.rb

```diff
@@ -3,3 +3,7 @@
 Given("the date is year: {int}, month: {int}, day: {int}") do |year, month, day|
   travel_to Time.zone.local(year, month, day)
 end
+
+Given("the date is year: {int}, month: {int}, day: {int}, hour: {int}") do |year, month, day, hour|
```

Comment:
Can we try and make this slightly more natural? Something like "Given the time is 14h00 on the 4/12/2021"?

Reply:
> Also is there a better formatting than `[Author name in brackets]`? Should be italics, or bold, or just... plain.

What I interpreted from the designs. But i'll discuss with Tom.